Adding content for Flink table and how to manage tables in applications (#1705)
aiven/devportal: docs/products/flink/howto/manage-flink-tables.rst

Manage tables in Apache Flink® applications
===========================================

> Aiven for Apache Flink® allows you to map source and target data structures as `Flink tables <https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/create/#create-table>`_ and use transformation statements to transform data. Some of the operations you can perform on a table include:

Reviewer: ```suggestion
Aiven for Apache Flink® allows you to map source and target data structures as `Flink tables <https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/sql/create/#create-table>`_ and use transformation statements to reshape, filter or aggregate data. Some of the operations you can perform on a table include:
```

* Import existing tables
* Add new tables

> * Clone tables

Reviewer: ```suggestion
* Clone table definitions from other applications
```

* Edit tables
* Delete tables

> Before performing any operation on a table in a Flink application, you must stop the application. To stop an application, navigate to the **Applications** tab on your Aiven for Apache Flink® service, select the desired application from the list, and click **Stop Deployment**.

Reviewer: is this true @HelenMel ? Can't you edit while an app is running?

Author: > is this true @HelenMel ? Can't you edit while an app is running?

At least when I tried on an application that is running, you can only view the table. You cannot edit. To edit or make changes, I had to stop the application and create a new version (I need to mention this in the doc.)
But @HelenMel, it would be great if you could confirm this. Thanks.

Add new table
--------------

Follow these steps add a new table to an application: 

> 1. Open the application to which you want to add a new table.

Reviewer: I know it's coming, but it would be good to reference the "application" concept document when we mention the app

2. Navigate to the **Add source tables** or **Add sink tables** screen within your application.
3. Click the **Add new table** button to add a new table to your application.

>     .. note:: 

Reviewer: The first `.` should be in line with the `Click` on the point above to be correctly displayed
```suggestion
   .. note:: 
```

>         If you already have a sink table listed, you must delete it before adding a new one.

Reviewer: ```suggestion
        If you already have a sink table listed, you must delete it before adding a new one, only one sink table is allowed per job.
```

4. Select the **Integrated service** from the drop-down list in the **Add new source table** or **Add new sink table** screen, respectively.
5. In the **Table SQL** section, enter the statement that will create the table. The interactive query feature if the editor will prompt you for error or invalid queries. 
6. Click **Add table** to complete the process.

Import an existing table
-------------------------
Follow these steps import an existing table from another application: 

1. In the **Add source tables** or **Add sink tables** screen, click **Import existing table** to import a table to your application. 
    .. note::
        If you already have a sink table listed, you must delete it before importing a new one.

2. From the **Import existing source table** or **Import existing sink table** screen:

    - Select the application from which you want to import the table.

>     -  Select the version of the application.

Reviewer: ```suggestion
    - Select the version of the application.
```

    - Select the table you want to import. 

3. Click **Next**.
4. Verify the data on the **Add new source table** or **Add new sink table** screen and click **Add table** to complete the process.

Clone table
-----------

> Follow these steps to clone a table in an application: 

Reviewer: ```suggestion
Follow these steps to clone a table within an application: 
```
